wkcuber/downsampling.py

```diff
@@ -462,10 +474,15 @@ def downsample_mag(
 ):
     interpolation_mode = parse_interpolation_mode(interpolation_mode, layer_name)
 
-    source_wkw_info = WkwDatasetInfo(path, layer_name, None, source_mag.to_layer_name())
+    source_wkw_info = WkwDatasetInfo(
+        path, layer_name, source_mag.to_layer_name(), wkw.Header(np.uint8)
+    )
```

Comment:
Previously the `voxel_type` was `None`. I am not sure why this should ever be `None` so I changed it to `uint8`. Is that correct?

Reply:
Hmm, semantically, it's not 100% correct. The thing is, that we don't know which voxel_type is actually used in the `source_wkw` info before opening the dataset. That's why, it wasn't specified before. Using `uint8` is merely a guess, which will work, since wkw will ignore it, but it's not really correct.
My suggestion would be as follows: Instead of providing a `wkw.Header` to `WkwDatasetInfo` simply pass `None`. In `ensure_wkw` you can assert that the header must not be `None` (since it's needed for the creation of a dataset).

So, the rule of thumb would be to only pass a header when the dataset might need to be created. If it's already there, don't pass a header. Does this make sense?